app/models/post.rb

```diff
@@ -1,3 +1,13 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id', counter_cache: true
+  has_many :likes
+  has_many :comments
+
+  def update_posts_count
```

Comment:
**You did great job by implementing the changes that given by previous reviewer.**

[OPTIONAL] I would like to give some feedbacks on your code. It would be nice if you use `after_save` method for updating posts count and make `update_posts_count` private. More info: [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html)